openquake/hazardlib/gsim/bindi_2014scaled.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Module exports :class:`BindiEtAl2014RhypEC8scaled`
+"""
+import numpy as np
+from openquake.hazardlib.gsim.bindi_2014 import BindiEtAl2014Rhyp, CoeffsTable
+
+
+class BindiEtAl2014RhypEC8scaled(BindiEtAl2014Rhyp):
+    """
+    Implements scaled European GMPE:
+    D.Bindi, M. Massa, L.Luzi, G. Ameri, F. Pacor, R.Puglia and P. Augliera
+    (2014), "Pan-European ground motion prediction equations for the
+    average horizontal component of PGA, PGV and 5 %-damped PSA at spectral
+    periods of up to 3.0 s using the RESORCE dataset", Bulletin of
+    Earthquake Engineering, 12(1), 391 - 340
+
+    The regressions are developed considering the geometrical mean of the
+    as-recorded horizontal components
+    The printed version of the GMPE was corrected by Erratum:
+    D.Bindi, M. Massa, L.Luzi, G. Ameri, F. Pacor, R.Puglia and P. Augliera
+    (2014), "Erratum to Pan-European ground motion prediction equations for the
+    average horizontal component of PGA, PGV and 5 %-damped PSA at spectral
+    periods of up to 3.0 s using the RESORCE dataset", Bulletin of
+    Earthquake Engineering, 12(1), 431 - 448. The erratum notes that the
+    printed coefficients tables were in error. In this implementation
+    coefficients tables were taken from the Electronic Supplementary
+    material of the original paper, which are indicated as being unaffected.
+    """
+    def _get_site_amplification_term(self, C, vs30):
```

Comment:
This method is identical to https://github.com/gem/oq-engine/blob/master/openquake/hazardlib/gsim/bindi_2014.py#L234 so I would remove it

Reply:
Removed